examples/misc/lib/effective_dart/style_bad.dart

```diff
@@ -10,3 +10,10 @@ class Dice {
   static final NUMBER_GENERATOR = new Random();
 }
 // #enddocregion const-names
+
+oneLineIf(dynamic overflowChars, dynamic other) {
+  // #docregion one-line-if-wrap
+  if (overflowChars != other.overflowChars)
+    return overflowChars < other.overflowChars;
+  // #enddocregion one-line-if-wrap
+};
```

Comment:
I think that the formatter is going to want a newline here ... if not, I'd suggest adding one :)

Reply:
It's actually a straight-up syntax error since this is at the top level. Oops! Fixed.